StructureMap/StructureMappingService.cs

```diff
@@ -10,7 +11,7 @@ public class StructureMappingService : IMessagingService
     {
         public string GetMessage()
         {
-            return "Hello from Structure Map!!!";
+            return ("Hello from Structure Map!!!");
```

Comment:
Where you have called this method, using dependency injection

Reply:
I called this method in Homecontroller to display a message in View